egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -844,7 +856,19 @@ if ! "${skip_train}"; then
     fi
 
 
+    mkdir -p ${ngram_exp}
     if [ ${stage} -le 9 ] && [ ${stop_stage} -ge 9 ]; then
+        if "${use_ngram}"; then
+            log "Stage 9: Ngram Training: train_set=${data_feats}/lm_train.txt"
+            cut -f 2 -d " " ${data_feats}/lm_train.txt | lmplz -S "20%" --discount_fallback -o ${ngram_num} - >${ngram_exp}/${ngram_num}gram.arpa
+            build_binary -s ${ngram_exp}/${ngram_num}gram.arpa ${ngram_exp}/${ngram_num}gram.bin 
+        else
+            log "Stage 9: Skip ngram stages: use_ngram=${use_ngram}"
```

Comment:
Can you increase this `--stop-stage 9` to 10? https://github.com/espnet/espnet/blob/586eb99492210f706a7c6cfc372a769485471b48/ci/test_integration.sh#L227
@kamo-naoyuki do you know anything else depending on the stage numbers >= 9 in espnet2?